### PR TITLE
When talk isn't configured log api attempts

### PR DESCRIPTION
**Pull request by edpaget**
There's no production talk at the moment, so just log that we tried to talk (heh) to it and do nothing.

### CODE REVIEW

File: app/workers/talk_admin_create_worker.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@ class TalkAdminCreateWorker
 
   def perform(project_id)
     Project.find(project_id).create_talk_admin(client)
+  rescue TalkApiClient::NoTalkHostError => e
+    Rails.logger.info e
```
Comment (camallen) [inline app/workers/talk_admin_create_worker.rb]: Can you log an error in honey badger as well just so we know this is happening outside of the logs? As per [here](https://github.com/zooniverse/Panoptes/blob/db3030d0c4276f0668fab93f61eb3e41ae6f8564/app/controllers/api/events_controller.rb#L60)

Comment (edpaget) [inline app/workers/talk_admin_create_worker.rb]: I can, but that's just going to clutter up honeybadger with notifications. We can stop rescuing this error when all the Talk environments are deployed and it'll go through to Honeybadger normally.

Comment (camallen) [inline app/workers/talk_admin_create_worker.rb]: Fair call, Let's just remember to rip it back out.

Comment (edpaget) [inline app/workers/talk_admin_create_worker.rb]: I'll add a note

**Pull request merged**